deployments/server/templates/ingress.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "inference-manager-server.fullname" . }}-grpc
+  labels:
+    {{- include "inference-manager-server.labels" . | nindent 4 }}
+spec:
+  ingressClassName: {{ .Values.global.ingress.ingressClassName }}
+  rules:
+  - http:
+      paths:
+      - path: /v1/chat/completions
```

Comment:
The gRPC requires a different ingress definition. Let's remove this ingress if we don't need now.